client/lib/helpers.js

```diff
@@ -38,6 +38,14 @@ export const linesToConfig = function(lines){
   return config;
 }
 
+// Initialises and returns an empty config object
```

Comment:
I'm thinking of coming back to this and encapsulating the config functions in a class rather than use helper functions. However, for now, the current status quo is to use helper functions and this will solve the immediate bug/issue.